experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -44,23 +46,28 @@ class Flowables {
   }
 
   template <typename T>
-  static Reference<Flowable<T>> just(std::initializer_list<T> list) {
-    auto lambda = [ list, it = list.begin() ](
+  static Reference<Flowable<T>> justN(std::initializer_list<T> list) {
+    std::vector<T> vec;
+    std::move(list.begin(), list.end(), std::back_inserter(vec));
+
+    size_t i = 0;
```

Comment:
Ah okay, it's implicit state. That makes sense, but it's non-obvious from a reading of the code. 

If the simple flowables are intended partially to teach people how to write their own flowables then perhaps a comment?